ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -3580,7 +3596,17 @@ public boolean dropPartition(String dbName, String tableName, List<String> parti
     List<String> pvals = MetaStoreUtils.getPvals(t.getPartCols(), partSpec);
 
     try {
-      names = getMSC().listPartitionNames(dbName, tblName, pvals, max);
+      GetPartitionNamesPsRequest req = new GetPartitionNamesPsRequest();
```

Comment:
I just scanned for get_partition methods that are used in common flows. Will pick up remaining ones in separate patches. This is just to avoid dealing with hundreds of test failures which just delays the entire patch.